setup/install/providers/aws/aws-ec2.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: single
+title:  "Amazon EC2"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+In [AWS](https://aws.amazon.com/){:target="\_blank"}, an [__Account__](/concepts/providers/#accounts)
+maps to a credential able to authenticate against a given [AWS
+account](https://aws.amazon.com/account/){:target="\_blank"}.
+
+## Option-1 : Use AWS Console to configure AWS
+
+Use this option to deploy Spinnaker, if you are familar with deployment using [AWS Console](https://console.aws.amazon.com/) .
+
+### Managing Account
+1. Navigate to [Console](https://console.aws.amazon.com/){:target="\_blank"} > CloudFormation and [select](https://docs.aws.amazon.com/awsconsolehelpdocs/latest/gsg/getting-started.html#select-region) your preferred region.
+2. Download [the template](https://github.com/spinnaker.github.io/blob/master/setup/install/providers/aws/managing.yaml) locally to your workstation.
+    
+    2.a (Optional). Add Managing account as shown [here](https://github.com/spinnaker.github.io/blob/master/setup/install/providers/aws/managing.yaml#L104)
+3. Creating the CloudFormation Stack
+    * __Create Stack__ > __Upload a template to Amazon S3__ > __Browse to template you downloaded in Step-2 above__ > __Next__  
+    * Enter __Stack Name__ as spinnaker-__managing__-infrastructure-setup and follow the prompts on screen to create the stack
+4. Once the stack is select the stack you created in Step-3 > Outputs and note the values. You will need these values for subsequent configurations.
+
+
+### In each of the Managing Account
+
+> These steps need to be carried out for the managing account as well.
+
+1. Navigate to [Console](https://console.aws.amazon.com/){:target="\_blank"} > CloudFormation and [select](https://docs.aws.amazon.com/awsconsolehelpdocs/latest/gsg/getting-started.html#select-region) your preferred region.
+2. Download [the template](https://github.com/spinnaker.github.io/blob/master/setup/install/providers/aws/managed.yaml) locally to your workstation.
+3. Creating the CloudFormation Stack
+    * __Create Stack__ > __Upload a template to Amazon S3__ > __Browse to template you downloaded in Step-2 above__ > __Next__  
+    * Enter __Stack Name__ as spinnaker-__managed__-infrastructure-setup and follow the prompts on screen to create the stack
+    * Enter __AuthArn__ and __ManagingAccountId__ as the value noted above and follow the prompts on screen to create the stack
+
+![](../outputs_cloudformation.png)
```

Comment:
I would either completely black out the rest of these credentials, or doubly confirm that these are not active credentials. With what is currently whited out you can reconstruct the keys.